groth16/src/generator.rs

```diff
@@ -86,34 +86,22 @@ impl<E: PairingEngine> ConstraintSystem<E::Fr> for KeypairAssembly<E> {
         LB: FnOnce(LinearCombination<E::Fr>) -> LinearCombination<E::Fr>,
         LC: FnOnce(LinearCombination<E::Fr>) -> LinearCombination<E::Fr>,
     {
-        fn eval<E: PairingEngine>(
```

Comment:
This was never really a polynomial evaluation so I moved and renamed it, so that we can share functionality between the prover/generator files